src/libs/actions/Card.ts

```diff
@@ -37,7 +37,8 @@ type IssueNewCardFlowData = {
     data?: Partial<IssueNewCardData>;
 };
 
-function reportVirtualExpensifyCardFraud(cardID: number) {
+function reportVirtualExpensifyCardFraud(card?: Card) {
+    const cardID = card?.cardID ?? -1;
```

Comment:
I m still not sure what is the best way to handle these. I feel like that if the cardID is not available we should cut this earlier, but not sure about the best practice here